javascript-local-storage/data.js

```diff
@@ -0,0 +1,8 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+
+if (previousTodosJSON !== null) {
+  todos = JSON.parse(previousTodosJSON);
+}
```

Comment:
Be sure to also listen for `'beforeunload'` to stringify and store the `todos` in local storage.

Reply:
![javascript-local-storage-02](https://user-images.githubusercontent.com/96809909/153800119-9490640b-79a7-4e53-ae61-3ad85e605d3d.gif)